src/librustc_error_codes/error_codes/E0696.md

```diff
@@ -0,0 +1,52 @@
+A function is using `continue` keyword incorrectly.
+
+Erroneous code example:
+
+```compile_fail,E0696
+// Simple continue pointing to an unlabeled break should yield in an error
```

Comment:
Please wrap the keywords between "\`", otherwise it's a bit complicated to read.

Reply:
@GuillaumeGomez 
like "// Simple continue pointing to an unlabeled break should yield in an error"
or   '// Simple continue pointing to an unlabeled break should yield in an error'
or   "Simple continue pointing to an unlabeled break should yield in an error"
or    '// Simple continue pointing to an unlabeled break should yield in an error'

Reply:
Like:

```rust
// Simple `continue` pointing to an unlabeled `break` should `yield` in an error
```